Hypnotic-Herons/sprint-3
# Db image

## Link to Ticket
#11 

## Description of Proposed Changes
user can upload an image that they cannot see

## Steps to Test

```
git fetch --all
git checkout db_image
code .
pip install pillow

```

1. delete migrations
1. python manage.py makemigrations website
1. python manage.py migrate
1. python manage.py runserver
1. log in http://127.0.0.1:8000
1. access the product view http://127.0.0.1:8000/product
1. add a product image upload should now be available
